### DISCUSSION TITLE
having issues running netflix lately, any tips?

### DISCUSSION THREAD

**Discussion by unevoljitelj** [Joshua-Riek/ubuntu-rockchip#382] [Q&A]
dunno if anything changed recently but its either agent string i am getting wrong or theres something with widevine but it doesnt work for me. It could be totaly me tho :P but if you have it working please share your settings/setup

dunno whats with armbian lately also, imposible to make it work, chromium not working properly.  using these two (ubuntu and armbian) but lately getting really frustrated with small things changing and making things fail. and i am not sure whats changing or why...

anyway, thanks :)

**Comment by Joshua-Riek** [comment]
Try running chromium with this command.
```
chromium-browser --user-agent="Mozilla/5.0 (X11; CrOS aarch64 15236.80.0) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/110.0.0.0 Safari/537.36"
```
